### PR TITLE
(Awaiting release) DOCS-3613: Document Exact Online automated integration

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/accounting/accounting-integrations.md [hunk 1]
```diff
@@ -94,14 +92,69 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
     - Your accounting package can process MT940 files.
     - The order numbers in the MT940 files also appear in the correct invoice fields in your accounting platform.
     - The customer's name and order amounts that appear in the exported MT940 files match your accounting platform.
+---
+</details>
 
+### Automated exports
+
+<details id="how-to-automate-exports-to-exact-online">
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:100]:
```suggestion
<details id="how-to-automate-exports-with-exact-online">
```

File: content/accounting/accounting-integrations.md [hunk 2]
```diff
@@ -94,14 +92,69 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
+<summary>How to automate exports with Exact Online</summary>
+<br>
+
+To automate exports to Exact Online, follow these steps:
+
+1. Grant MultiSafepay permission to access your Exact Online account:
+   
+   1. Sign in to your <a href="https://merchant.multisafepay.com" target="_blank">MultiSafepay dashboard</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:108]: Should we use roman numerals to help differentiate between the two numbering sets?

File: content/accounting/accounting-integrations.md [hunk 3]
```diff
@@ -94,14 +92,69 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
+
+2. Map MultiSafepay transaction classes to Exact Online account ledgers:
+   
+   1.  In your MultiSafepay dashboard, a list MultiSafepay transaction classes and Exact Online account ledgers appears.
+   2.  Sign in to Exact Online again, and go to **Financial** > **Entries & G/L accounts** > **G/L accounts** > **Create**.
+   3.  From your dashboard, copy the first account ledger name.
+   4.  In Exact Online:
+       - In the **Code** field, paste in the account ledger name.
+       - In the **Description** field, enter a description of the account ledger.
+       - From the **Type** list, select the type: **90 "General"** for all except **MultiSafepayImportedRows**, which is **12 "Bank"**).
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:128]:
```suggestion
       - From the **Type** list, select the type: **90 "General"** for all except **MultiSafepayImportedRows**, which is **12 "Bank"**.
```

File: content/accounting/accounting-integrations.md [hunk 4]
```diff
@@ -189,12 +218,10 @@ For each new invoice in Exact, if you:
 
 - **Email:** The customer receives a second email containing a payment link, in addition to the normal email.
 - **Print and process:** The customer receives an email containing a payment link.
-
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:221]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md [hunk 5]
```diff
@@ -217,6 +244,7 @@ xCore offers two reconciliation apps for Exact accounting tools. The apps automa
 
 For more information, see xCore – <a href="https://xcore.nl/afletteren-2-0-exact-online-multisafepay/" target="_blank">Afletteren 2.0 Exact Online with MultiSafepay</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
 
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:247]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md [hunk 6]
```diff
@@ -319,7 +345,7 @@ You must first provide Twinfield with an additional international bank account n
 4. In the **Account number** field, enter any 7 digits.
 5. Click **Calculate IBAN**.  
     A dummy IBAN and BIC code are generated.
-
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:348]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md [hunk 7]
```diff
@@ -337,16 +363,16 @@ You must first provide Twinfield with an additional international bank account n
 - **IBAN**
 7. In **Cash & Banks**, select **Drag and drop bank statements** or **Browse**.
 8. Upload the MultiSafepay MT940 file.
-
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:366]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md [hunk 8]
```diff
@@ -389,6 +415,7 @@ For import instructions, see Visma – <a href="https://nl.visma.com/accountview
 
 For more information, see Yuki - <a href="https://support.yuki.nl/en/support/solutions/articles/80000786497-upload-files-from-po-box" target="_blank">Upload files from PO box</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
 
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:418]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md [hunk 9]
```diff
@@ -400,11 +427,10 @@ For more information, see Yuki - <a href="https://support.yuki.nl/en/support/sol
 
 For more information, see Yuki - <a href="https://support.yuki.nl/nl/support/solutions/articles/80000787813-bank-verwerkingsregel-aanmaken" target="_blank">Create bank processing rule</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
 
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:430]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md [hunk 10]
```diff
@@ -416,6 +442,7 @@ To connect Duopact with your MultiSafepay account, see Duopact – <a href="http
 
 Provide Duopact with your MultiSafepay [site API key](/docs/sites#site-id-api-key-and-security-code). They will set up a Yuki account for you. 
 
+---
```
Comment (Sarah-MultiSafepay) [inline content/accounting/accounting-integrations.md:445]:
```suggestion
---

```

File: content/accounting/accounting-integrations.md (hunk 2 above)
Comment (FrancescaNCook) [reply content/accounting/accounting-integrations.md:108]: @Sarah-MultiSafepay Readme does that automatically, but I'm happy to make the markdown match.

Comment (FrancescaNCook) [reply content/accounting/accounting-integrations.md:108]: Hmm, the markdown doesn't seem to like it. I think I'll leave it as is in the backend. It renders nicely in Readme.

**Review by Sarah-MultiSafepay — APPROVED** [review]

File: wordlist.txt [hunk 11]
```diff
@@ -151,6 +151,7 @@ Edenred's
 Edoburu
 Electronische
 Electrosex
+favicon
```
Comment (Sarah-MultiSafepay) [inline wordlist.txt:154]:
```suggestion
favicon
MultiSafepayImportedRows
```

**Review by Rendani-MultiSafepay — APPROVED** [review]

File: content/accounting/accounting-integrations.md [hunk 12]
```diff
@@ -90,31 +88,89 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
+   
+1. Sign in to your <a href="https://merchant.multisafepay.com" target="_blank">MultiSafepay dashboard</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
+2. Go to **Integrations** > **Accounting** > **Exact Online** > **Configure**.
+    You are redirected to Exact Online. 
+3. Enter your email address and Exact Online password, and then click **Next**.
+    A list of required permissions is displayed.
+4. Grant MultiSafepay access to Exact Online only or to Exact Online and future companies. 
+5. Select the terms and conditions and risk disclaimer checkboxes to accept them.
+6. Click **Allow**.
+    You are redirected to your MultiSafepay dashboard. This make take a few seconds while MultiSafepay creates ledger accounts for you.
```
Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:114]: may take

File: content/accounting/accounting-integrations.md [hunk 13]
```diff
@@ -90,31 +88,89 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
+<br>
+
+To grant MultiSafepay permission to access your Exact Online account:
+   
+1. Sign in to your <a href="https://merchant.multisafepay.com" target="_blank">MultiSafepay dashboard</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
+2. Go to **Integrations** > **Accounting** > **Exact Online** > **Configure**.
+    You are redirected to Exact Online. 
+3. Enter your email address and Exact Online password, and then click **Next**.
+    A list of required permissions is displayed.
+4. Grant MultiSafepay access to Exact Online only or to Exact Online and future companies.
```
Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:111]: further?

File: content/accounting/accounting-integrations.md [hunk 14]
```diff
@@ -90,31 +88,89 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
+    You are redirected to Exact Online. 
+3. Enter your email address and Exact Online password, and then click **Next**.
+    A list of required permissions is displayed.
+4. Grant MultiSafepay access to Exact Online only or to Exact Online and future companies. 
+5. Select the terms and conditions and risk disclaimer checkboxes to accept them.
+6. Click **Allow**.
+    You are redirected to your MultiSafepay dashboard. This make take a few seconds while MultiSafepay creates ledger accounts for you. 
+    ✅ &nbsp; You have successfully granted MultiSafepay access permissions.
+
+❗️If you rename, modify, or delete information in Exact Online, exports will fail. If you get an error, email <support@multisafepay.com>
```
Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:117]: If you encounter

Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:117]: space after excl point

**Review by inesmayerMSP — CHANGES_REQUESTED** [review]

File: content/accounting/accounting-integrations.md [hunk 15]
```diff
@@ -90,31 +88,71 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
+    You are redirected to Exact Online. 
+3. Enter your email address and Exact Online password, and then click **Next**.
+    A list of required permissions is displayed.
+4. Grant MultiSafepay access to Exact Online only or Exact Online and future companies. 
+5. Select the checkbox to accept the Exact Online terms and conditions.
+6. Click **Allow**.
+    You are redirected to your MultiSafepay dashboard. This may take a few seconds while MultiSafepay creates ledger accounts for you. 
+    ✅ &nbsp; You have successfully granted MultiSafepay access permissions.
+
+❗️If you delete, or modify information in Exact Online, exports will fail. If you get an error, email <support@multisafepay.com>
```
Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:117]: if you delete or modify; If you encounter an error

File: content/accounting/accounting-integrations.md [hunk 16]
```diff
@@ -171,26 +193,36 @@ To connect your MultiSafepay account to e-Boekhouden, see PSP betalingen – <a
...
 
+<details id="how-to-import-manually">
+<summary>How to add a financial year </summary>
 <br>
 
-## Denovit integration
+1. Sign in to your Exact Online account.
+2. On the navigation menu > click  **sample company Exact online** > **Master Data**.
+3. Under **Financial** > select **Period-data table**.
+4. Under **Financial years** page > click on **New** >  **Create** <br> Create new financial year pop-up screen appears.
```
Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:214]: needs to be coherent: Under xxxx page / under the xxx page / under xxx

File: content/accounting/accounting-integrations.md (hunk 15 above)
Comment (inesmayerMSP) [inline content/accounting/accounting-integrations.md:117]: if you delete or modify; if you encounter an error,

**Review by inesmayerMSP — CHANGES_REQUESTED** [review]

**Pull request merged**